clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala

```diff
@@ -16,6 +22,18 @@ abstract class EnvIntegrationSpec(env: String)
     extends BaseIntegrationSpec(s"Clio in $env")
     with IntegrationSuite {
 
+  /** URL of vault server to use when getting bearer tokens for service accounts. */
+  private val vaultUrl = "https://clotho.broadinstitute.org:8200/"
```

Comment:
The integration test code still uses the vault client because it felt simpler than trying to auto-generate the account JSON at test-runtime using SBT, and everywhere we run the ITs has access to vault / a vault token.